server.py

```diff
@@ -3,13 +3,13 @@
 inputs = []
 outputs = []
 message_queues = {}
-
+data_to_send = 0
```

Comment:
Using a global variable for this kind of code could be awful if you ever have to make it multi threaded, there are better solutions to this

Reply:
fixed